src/App.js

```diff
@@ -1,29 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```

Comment:
- [ ] As per the guideline, The `App.js` component should be class-based. As we are learning to implement the class-based component in this milestone.